### PR TITLE
Update bundler to v2.1.4

### PR DESCRIPTION
**Pull request by larouxn**
I don't quite recall why we locked Bundle to v1.17.2 but that's from December 2018. We can/should probably use a fresher version. :)

bundler

```
/nix/store/lcabxws0khz10wz0614pxqz67fjhkm78-ruby-2.5.3/lib/ruby/2.5.0/rubygems/dependency.rb:312:in `to_specs': Could not find 'bundler' (2.1.4) required by your /home/travis/build/robwhitaker/mmp-website/Gemfile.lock. (Gem::MissingSpecVersionError)
```

Seems `gem install bundler; bundle update --bundler` could resolve this.

---

But then the build passed anyway. 🤷‍♂️
Screen Shot 2020-01-18 at 22 44 24

### CODE REVIEW

**Review by robwhitaker — CHANGES_REQUESTED** [review]
We locked Bundle mostly as a side-effect of it being managed by Nix. That's just the version of Bundle we got with the package set we pinned. That said, you can't just upgrade Bundle via the usual route (in this case, `bundle --update bundler`). That might update your local copy of Bundle, but our Nix environment still tries to used the pinned one.

Right now, the build technically exits without error, but there are still errors due to updating the `Gemfile.lock` without updating the Nix derivation:

image

Basically, we need to update the Nix package set we're getting our Ruby dependencies from.

**Comment by larouxn** [comment]
Ah, I see. I can't seem to find a Bundler > v2.0 package on [nixos.org](https://nixos.org/nixos/packages.html?channel=nixpkgs-unstable&query=bundler). 🤔

**Comment by larouxn** [comment]
Closing as this does not work. We need to wait for the Nix Bundler 2.x package.
